internal/client.go

```diff
@@ -94,6 +94,17 @@ type (
 		//	- InternalServiceError
 		SignalWorkflow(ctx context.Context, workflowID string, runID string, signalName string, arg interface{}) error
 
+		// SignalWithStartWorkflow sends a signals to a workflow in execution, if workflow is not running or not found,
+		// it starts workflow then sends the signal in transaction.
+		// - workflowID, runID, signalName, signalArg are same as SignalWorkflow's parameters
+		// - options, workflow, workflowArgs are same as StartWorkflow's parameters
```

Comment:
suggested update:
The options, workflow, and workflowArgs parameters are same as those of StartWorkflow.